src/index.html

```diff
@@ -7,6 +7,479 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#samsung"
+              class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#smartphones"
+              class="nav__link">
```

Comment:
```suggestion
              class="nav__link"
            >
```
See my previous comments